src/app/Unauthenticated.tsx

```diff
@@ -1,7 +1,53 @@
-import Login from 'pages/Login';
+import { Card, CardContent, Grid, Typography } from '@mui/material';
+import { Auth } from '@supabase/ui';
+import Topbar from 'components/header/Topbar';
+import { FormattedMessage } from 'react-intl';
+import { supabase } from 'services/supabase';
+import { getLoginSettings } from 'utils/env-utils';
 
 const Unauthenticated = () => {
-    return <Login />;
+    const loginSettings = getLoginSettings();
+
+    return (
+        <Grid
+            container
+            spacing={0}
+            direction="column"
+            sx={{
+                alignItems: 'center',
+                height: '100vh',
+                justifyContent: 'center',
+            }}
+        >
+            <Topbar isNavigationOpen={false} />
```

Comment:
It is my understanding that a `Grid` container should always wrap `Grid` items. Therefore, should `Topbar` be encased in a `Grid` item?

Reply:
We could... doesn't impact the UI from what I can tell.